admin_ui/src/components/KeySearch.vue

```diff
@@ -42,18 +40,19 @@
 </template>
 
 <script lang="ts">
+import { defineComponent } from "vue"
 import { FetchIdsConfig } from "../interfaces"
 
-export default {
+export default defineComponent({
     props: {
         fieldName: String,
         tableName: String,
-        rowID: undefined,
-        readable: undefined,
+        rowID: undefined as any,
+        readable: undefined as any,
     },
     data() {
         return {
-            ids: [],
+            ids: [] as any,
```

Comment:
`number[]` might work here

Reply:
Unfortunately `number[]` does not work. `[] as number[]` or `Array<number>` also does not work